cmd/testscript/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"clickhouse-playground/cmd/testscript/playgroundclient"
+	"clickhouse-playground/cmd/testscript/processor"
+	"flag"
+	"fmt"
+	zlog "github.com/rs/zerolog/log"
+	"time"
+)
+
+type Action string
+
+const (
+	RunTest    Action = "run-test"
+	ImportRuns        = "import-runs"
+)
+
+func main() {
+	var action string
+	var needHelp bool
+	var importRunsBefore string
+	var importRunsAfter string
+	var importResultFilePath string
+
+	flag.BoolVar(&needHelp, "help", false, "help")
+	flag.StringVar(&action, "action", "", "action to process: run-test, import-runs")
+	flag.StringVar(&importRunsBefore, "import-runs-before", "", "border for runs import from AWS. Format 2006-01-02 15:04:05")
```

Comment:
The intended action is specified in the `action` flag, so we can omit the `import-runs-` prefix from the `before` and `after` arguments.